src/Button.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+export default function Button({ handleClick, children }) {
+  return <StyledButton onClick={handleClick}>{children}</StyledButton>;
+}
+
+const StyledButton = styled.button`
```

Comment:
I think it should work as well if you just export the `const StyledButton` so that you do not need the function here.

Reply:
I see, will try that. Thx.

Reply:
I think so, but I'm not sure about this yet!